website/docs/docs/dbt-versions/core-versions.md

```diff
@@ -8,6 +8,14 @@ pagination_prev: null
 
 dbt Core releases follow [semantic versioning](https://semver.org/) guidelines. For more on how we use semantic versions, see [How dbt Core uses semantic versioning](#how-dbt-core-uses-semantic-versioning). 
 
+:::tip Keep on latest version, always
+
+_Did you know that you can always be working on the latest version?_ 
+
+With dbt Cloud, you can get early access to new functionality before it becomes available in dbt Core and without the need of managing your own version upgrades. Refer to the [Keep on latest version](/docs/dbt-versions/upgrade-core-in-cloud#keep-on-latest-version) setting for details.
+
+:::
```

Comment:
made text suggs here to keep it "evergreen" (vs upgrade v1.8 guide which is less evergreen).

Reply:
good by me!